src/logging.h

```diff
@@ -137,7 +137,7 @@ namespace BCLog {
         /** Returns a string with the log categories */
         std::string LogCategoriesString()
         {
-            return Join(LogCategoriesList(), ", ", [&](const LogCategory& i) { return i.category; });
+            return Join(LogCategoriesList(), std::string(", "), [&](const LogCategory& i) { return i.category; });
```

Comment:
Why is this one needed? (Sorry I missed it in my last review comment)

Reply:
Thank you. [Fixed](https://github.com/bitcoin/bitcoin/pull/18922#issuecomment-626343159).